### DISCUSSION TITLE
BioSteam for data collection

### DISCUSSION THREAD

**Discussion by adilsal33m** [BioSTEAMDevelopmentGroup/biosteam#161] [General]
I am exploring fermentation models and so far have found BioSteam to be fairly easy to use (I am not a chemical engineer). 

My end goal with these models is to generate simulation data over the process duration (I have read fermentation takes long time to complete). To be more precise, I am intending to introduce disturbances in the parameters during the process lifecycle to see how that would vary the output.

Can this be done through BioSteam?

**Comment by yoelcortes** [comment]
@adilsal33m, dynamic simulation capabilities are not yet supported in BioSTEAM. You can check out QSDsan, which has some dynamic simulation capabilities with an assumption on constant volumetric flow rate:

https://qsdsan.readthedocs.io/en/latest/

Thanks,
